io.catenax.digital_product_passport/1.0.0/DigitalProductPassport.ttl

```diff
@@ -0,0 +1,846 @@
+    bamm:description "The expected product lifetime (lifespan)."@en;
+    bamm:characteristic :LifespanCharacteristic.
+:LifespanCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Lifespan Characteristic"@en;
+    bamm:description "Characteristic for the expected product lifetime (lifespan)."@en;
+    bamm:dataType :LifespanEntity.
+:placedOnMarket a bamm:Property;
+    bamm:preferredName "Placed on Market"@en;
+    bamm:description "Date on  when the product is put in the market."@en;
+    bamm:characteristic :TimeStamp;
+    bamm:exampleValue "2000-01-01"^^xsd:date.
+:warranty a bamm:Property;
+    bamm:preferredName "Warranty"@en;
+    bamm:description "Commercial warranty period of the product (coupled to the Vehicle) in months. Durability and reliability of the product or its components as expressed through the product’s guaranteed lifetime, technical lifetime, mean time between failures, indication of real use information on the product, resistance to stresses or ageing mechanisms."@en;
+    bamm:characteristic :WarrantyCharacteristic;
+    bamm:exampleValue "36"^^xsd:decimal.
+:WarrantyCharacteristic a bamm-c:Measurement;
+    bamm:preferredName "Warranty Characteristic"@en;
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:month.
+:owner a bamm:Property;
+    bamm:preferredName "Owner"@en;
+    bamm:description "The owner of the product and the passport."@en;
+    bamm:characteristic :OwnerCharacteristic.
+:OwnerCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Owner Characteristic"@en;
+    bamm:description "Owner of the product and passport."@en;
+    bamm:dataType :OwnerEntity.
+:manufacturerId a bamm:Property;
+    bamm:preferredName "Manufacturer Id"@en;
+    bamm:description "The main manufacturer, if different from the passport owner."@en;
+    bamm:characteristic :CompanyIdentification;
+    bamm:exampleValue "BPN1234567890ZZ".
+:CompanyIdentification a bamm:Characteristic;
+    bamm:preferredName "Company Identification"@en;
+    bamm:description "Identification of the company or location of the company. In the CATENA-X network the BPN/L identification is needed."@en;
+    bamm:dataType xsd:string.
+:importer a bamm:Property;
+    bamm:preferredName "Importer"@en;
+    bamm:description "The importer of the product, if different from the owner of the passport."@en;
+    bamm:characteristic :CompanyIdentification;
+    bamm:exampleValue "BPNL1234567890ZZ".
+:gtin a bamm:Property;
+    bamm:preferredName "GTIN"@en;
+    bamm:description "GTIN stands for Global Trade Item Number. It is a unique and internationally recognized identifier used to identify products in the global marketplace. GTINs are typically encoded in barcodes and are used by retailers, manufacturers, and suppliers to track and manage their products throughout the supply chain. Valid inputs are all GTIN versions (GTIN-8, GTIN-12, GTIN-13, and GTIN-14)"@en;
+    bamm:characteristic :GtinTrait;
+    bamm:exampleValue "GTIN-8: 12345678".
```

Comment:
habe you checked if it validates against the regex?

Reply:
fixed example (deleted start)